ci/install_abseil.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+export DEBIAN_FRONTEND=noninteractive
+
+if ! type cmake > /dev/null; then
+    echo "cmake not installed. Exiting.." >&2
```

Comment:
Technically we do have at least two scripts that allow to install cmake if necessary:
- `/ci/setup_cmake.sh` - that's only Linux x86_64. But installs fast.
- `/tools/setup-cmake.sh` - that's everything. But it builds it from source. And it's building slowly.

I'd suggest you don't even check for cmake in here. Assume it's given.

Reply:
Sure will remove it.